### ISSUE TITLE
Custom package source options

### ISSUE THREAD

**Issue by colinkiama** [vanat/vanat#16]
1. Another central package repository like VPackagist
2. Getting packages directly from their repositories (like golang does with the `go get` command: https://pkg.go.dev/cmd/go#hdr-Add_dependencies_to_current_module_and_install_them)

**Comment by dar5hak** [comment]
The difficult part with the first option is to make sure library authors publish each update to the central repo. The `go get` approach seems more… approachable.

**Comment by colinkiama** [comment]
> The difficult part with the first option is to make sure library authors publish each update to the central repo.

That's intentional. The point of the 1st option is to provide an option to use other central repositories instead of relying on VPackagist.

Ideally the majority of projects would be on VPackagist however some packages may need to be private or hosted in a different place due to external requirements.

Maybe I'm thinking too far ahead with this? 😅

**Comment by colinkiama** [comment]
However, the 2nd option definitely needs to be added.

**Comment by dar5hak** [comment]
Agreed, people might want to point to a different repo. Although at the time of writing, that'd be a rare scenario. Just a guess, I could be wrong.
